app/models/tube_rack/purpose.rb

```diff
@@ -12,10 +14,10 @@ def self.standard_tube_rack
     TubeRack::Purpose.find_by(name: 'TR Stock 96')
   end
 
-  def create!(*args, &block)
+  def create!(*args, &)
```

Comment:
I struggle enough to understand what methods like this do (with blocks) without the linting further shortening it to an ampersand!